contracts/igp/interchain-gas-paymaster/src/interface.sw

```diff
@@ -0,0 +1,46 @@
+library interface;
+
+/// Logged when the benficiary is set.
+pub struct BeneficiarySetEvent {
+    new_beneficiary: Identity,
```

Comment:
not sure it matters but is it nice to keep the keys here 1:1 with solidity"?

Reply:
will change to `beneficiary` instead of `new_beneficiary`